cdap-ui/app/cdap/components/DataPrepHome/index.js

```diff
@@ -221,9 +226,9 @@ export default class DataPrepHome extends Component {
             type: DataPrepActions.disableLoading,
           });
           DataPrepStore.dispatch({
-            type: DataPrepActions.setDataError,
+            type: DataPrepActions.setWorkspaceError,
```

Comment:
Are we replacing all `setDataError` to `setWorkspaceError`? I now don't see this event being emitted anywhere.

Reply:
We're dispatching this when call to get workspace fails. We still use `setDataError` in other places like in `DataPrepActionCreator`